device/transport/amqp/src/amqp_twin_client.ts

```diff
@@ -92,11 +92,6 @@ export class AmqpTwinClient extends EventEmitter {
             debug('_handleError: error is: ' + err);
             this.emit(AmqpTwinClient.errorEvent, translateError('received an error from the amqp transport: ', err));
           },
-          handleLinkDetach: (detachObject: any) => {
-            if (detachObject && detachObject.error) {
-              this.emit(AmqpTwinClient.errorEvent, translateError('received an error from the amqp transport: ', detachObject.error));
-            }
-          },
```

Comment:
I'm thinking I just didn't put this in there for grins.  At least at the time that this was originally written, a detach bubbled up from below wouldn't necessary cause an error event to be emitted.  The upper clients are going to need those, yes?

Reply:
the base layer takes care of caching and managing link states

---
In reply to: [167288747](https://github.com/Azure/azure-iot-sdk-node/pull/228#discussion_r167288747) [](ancestors = 167288747)